crates/gpui/src/platform/mac/window.rs

```diff
@@ -1007,8 +1006,27 @@ impl PlatformWindow for MacWindow {
                 NSColor::clearColor(nil)
             };
             this.native_window.setBackgroundColor_(clear_color);
-            let window_number = this.native_window.windowNumber();
```

Comment:
The `this.native_window.setHasShadow_(opaque);` above should also be removed I think?